frontend/public/components/nav/perspective-nav.tsx

```diff
@@ -107,12 +107,12 @@ const PerspectiveNav: React.FC<{}> = () => {
       : [];
 
   return (
-    <div className="oc-perspective-nav">
+    <ul className="oc-perspective-nav pf-c-nav__list">
```

Comment:
I believe this will generate `ul > section > ul` inside the dev perspective causing the tests to fail.

Reply:
Converted this back to `<div>`